src/server/visual-diff-reporter.js

```diff
@@ -8,17 +8,25 @@ const __dirname = dirname(fileURLToPath(import.meta.url));
 
 function createData(rootDir, sessions) {
 
+	let metadata = {};
+	const metadataPath = join(rootDir, PATHS.METADATA);
+	if (existsSync(metadataPath)) {
+		metadata = JSON.parse(readFileSync(metadataPath));
+	}
+
 	const files = new Map();
 	const browsers = new Map();
 
 	sessions.forEach(s => {
 
 		const browserName = s.browser.name;
 		if (!browsers.has(browserName)) {
+			const prevBrowser = metadata.browsers?.find(b => b.name === browserName);
 			browsers.set(browserName, {
 				name: browserName,
 				numFailed: 0,
-				version: s.browser.browser.version().substring(0, s.browser.browser.version().indexOf('.'))
+				version: parseInt(s.browser.browser.version().substring(0, s.browser.browser.version().indexOf('.'))),
```

Comment:
I think `parseInt` is going to stop at the `.` anyway.
```suggestion
				version: parseInt(s.browser.browser.version()),
```

Reply:
That's a nice side benefit, thanks!